### ISSUE TITLE
Open class by name doesn't work on Server-Side editing

### ISSUE THREAD

**Issue by dgonzalez-intersystems** [intersystems-community/vscode-objectscript#915]
Hi,
For some time now, when working in Server-Side Editing, the option to open class by name (Ctrl+T for Windows or Command+T for MAC) doesn't work. It finds properties or methods within the classes that match the searched pattern, but not the classes with that name.
<img width="833" alt="image" src="https://user-images.githubusercontent.com/78598637/161509596-78a82605-412f-40ad-a65d-e007183e32ba.png">
<img width="789" alt="image" src="https://user-images.githubusercontent.com/78598637/161509708-bb150048-8ca8-4631-8434-d4fb513063b5.png">

Thanks!

**Comment by gjsjohnmurray** [comment]
Have you followed the README steps to enable proposed APIs? If this used to work for you, perhaps you previously di the config steps, downloaded the beta VSIX and installed it, but subsequently updated the extension to a newer non-beta version. In that case, download the -beta.1.vsix file whose version number is bigger than your current VSIX upgraded from Marketplace.

**Comment by dgonzalez-intersystems** [comment]
Hi John,
Thanks for your fast reply!
Yes, I followed these README steps. I have the last VSIX beta version (v1.4.5-beta.7), and I enabled proposed APIs as explained. In fact, the option to search in all files (what also needs this config to work) is working properly.

Here's my Output, my argv.json file and a screenshot showing how the "search in all files" option is working.
<img width="810" alt="image" src="https://user-images.githubusercontent.com/78598637/161527364-cffc4156-588a-45d4-82d0-482c76c74bba.png">

<img width="756" alt="image" src="https://user-images.githubusercontent.com/78598637/161527506-784c9ded-61e3-41c4-8523-46d192425711.png">

<img width="514" alt="image" src="https://user-images.githubusercontent.com/78598637/161527690-ac9bd5fd-6ce6-4af2-aec7-47ea16104dc2.png">

Thanks!

**Comment by gjsjohnmurray** [comment]
Do you find the classes if you use Ctrl+P or Cmd+P instead? Or alternatively delete the # prefix in the Ctrl/Cmd+T field before you type the class name?

**Comment by dgonzalez-intersystems** [comment]
With Command+P I only find those classes which are open in the editor right now or I've opened recently. But those classes that I've never opened or I haven't opened for some time are not found.

**Comment by dgonzalez-intersystems** [comment]
Hi again John,
Enabling the "Search > Quick Open: Include Symbols" option I can find the classes if I use Cmd+P
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/78598637/161546139-eb2734b0-9fac-454d-a29f-f3fd95e182b9.png">

It's not the same behaviour, because routines are not found either, but at least I find the classes.

**Comment by isc-bsaviano** [comment]
@dgonzalez-intersystems Yes, Ctrl-P (quick open) works for all files. Ctrl-T  is for looking up a specific symbol in a file (right now just class members). In the past, Ctrl-T did show classes as well (but not any other type of file). When I overhauled the `WorkspaceSymbolProvider` (which provides data for Ctrl-T) in #772, I removed the classes because they were already covered by Ctrl-P. We can consider showing classes in Ctrl-T again if that is desirable.
